apps/challenges/views.py

```diff
@@ -352,25 +352,6 @@ def challenge_phase_detail(request, challenge_pk, pk):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
-@throttle_classes([AnonRateThrottle])
```

Comment:
The PR remove the complete view function which should not be the case.
In order to dive into `pagination`, please read [here](http://www.django-rest-framework.org/api-guide/pagination/).